Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -5415,6 +5415,21 @@ NeedsStorageAccessFromFileURLsQuirk:
     WebCore:
       default: true
 
+NonPortaledModelEnabled:
```

Comment:
This flag is hard to read. It's Non-Portal'ed model.
How about ModelWithoutPortal or ModelWithPortalDisabled?

Reply:
Or maybe `NoPortalModelEnabled`?

Reply:
I'm leaning towards `ModelWithoutPortalEnabled` or `NoPortalModelEnabled`. cc @achan00

Reply:
I like `ModelWithoutPortalEnabled`

Reply:
`ModelWithoutPortal` may be simpler (worried whether there's confusion with how `ModelWithoutPortalEnabled` is parsed: ModelWithPortal-Enabled or ModelWith-PortalEnabled).

Reply:
Per offline discussion, we ended up using `ModelNoPortalAttributeEnabled`, as the name reflects its purpose clearly.